workspaces/frontend/src/app/context/NamespaceContextProvider.tsx

```diff
@@ -26,20 +29,24 @@ export const NamespaceContextProvider: React.FC<NamespaceContextProviderProps> =
   const [namespaces, setNamespaces] = useState<string[]>([]);
   const [selectedNamespace, setSelectedNamespace] = useState<string>('');
   const [namespacesData, loaded, loadError] = useNamespaces();
+  const [lastUsedNameSpace, setLastUsedNamespace] = useStorage<string>(storageKey, '');
```

Comment:
```suggestion
  const [lastUsedNamespace, setLastUsedNamespace] = useStorage<string>(storageKey, '');
```
Please also change its usages.